.github/workflows/integration-test.yml

```diff
@@ -0,0 +1,77 @@
+name: Integration Test
+
+on:
+  push:
+    branches: [master]
+
+jobs:
+  package:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Node and pnpm
+        uses: logto-io/actions-node-pnpm-run-steps@v1.2.1
+
+      - name: Package
+        run: ./package.sh
+
+      - uses: actions/upload-artifact@v3
+        with:
+          name: integration-test-${{ github.sha }}
+          path: /tmp/logto.tar.gz
+          retention-days: 3
+  
+  run-logto:
+    needs: package
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, ubuntu-18.04, macos-11, macos-12]
+    
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+
+      - name: Start Postgres (Linux)
+        if: runner.os == 'Linux'
+        run: |
+          sudo service postgresql start
+          sudo -u postgres psql -c "ALTER USER postgres PASSWORD 'postgres';"
+
+      - name: Start Postgres (macOS)
+        if: runner.os == 'macOS'
+        run: |
+          pg_ctl -D /usr/local/var/postgres start
+          psql postgres -c "CREATE USER postgres WITH SUPERUSER PASSWORD 'postgres';"
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: integration-test-${{ github.sha }}
+      
+      - name: Display structure of downloaded files
+        run: ls -R
+
+      - name: Extract
+        run: tar -xzf logto.tar.gz
+      
+      - name: Rebuild Argon2
+        run: npx node-pre-gyp rebuild -C .
+        working-directory: logto/packages/core/node_modules/argon2
+
+      - name: Run Logto
+        run: node . --from-root --all-yes &
+        working-directory: logto/packages/core
+        env:
+          NODE_ENV: production
+          DB_URL_DEFAULT: postgres://postgres:postgres@localhost:5432
+
+      - name: Sleep for 5 seconds
+        run: sleep 5
+
+      - name: Health check
+        run: curl http://localhost:3001/api/status -If
```

Comment:
What does the `-If` do?

Reply:
<img width="957" alt="image" src="https://user-images.githubusercontent.com/14722250/167528112-42f213d4-0422-4372-8650-d70fb4102099.png">

Reply:
So we do not want to output failures on this healthcheck API?

Reply:
<img width="395" alt="image" src="https://user-images.githubusercontent.com/14722250/167536121-5a4b47e4-37d9-4c57-9aef-aabf17a08a62.png">

this param is confusing, it can make the step to fail (exit with non-zero code), as we don't need the output.